cmd/platform.go

```diff
@@ -160,18 +164,21 @@ func createGithubClient(flag *flag.FlagSet, verifyFlags bool, readOnly bool) (mu
 	if err != nil {
 		return nil, err
 	}
-
+	repoIncludeFilterCompile, repoIncludeErr := regexp.Compile(repoIncludeFilter)
+	repoExcludeFilterCompile, repoExcludeErr := regexp.Compile(repoExcludeFilter)
```

Comment:
No need to send the error down just to check it and error it. Just error here instead.